.github/renovate.json5

```
@@ -71,4 +71,18 @@
     "dockerfile",
     "rpm",
   ],
+  "packageRules": [{
+    "matchPackageNames": ["/.*/"],
```

I know, Collector was done differently, but I'd still comment. There's a simpler (and a tiny bit more performant) way to write "match everything". See https://docs.renovatebot.com/string-pattern-matching/
```suggestion
    "matchPackageNames": ["*"],
```